views/partials/footer.hbs

```diff
@@ -7,6 +7,19 @@
                 class="mega-octicon octicon-mark-github vertical-middle"></span></a></li>
           <li><a class="footer-social-icon" href="https://twitter.com/electronjs" title="@electronjs on Twitter"><span
                 class="devicon-twitter-plain vertical-middle"></span></a></li>
+          <li><a class="footer-social-icon vertical-middle" href="https://discord.gg/H6uTh7m"
+              title="The Discord Server">
+              <span class="vertical-middle">
+              <svg class="footer-discord" height="32px" enable-background="new 0 0 512 512" version="1.1"
+                viewBox="0 0 512 512" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"
+                xmlns:xlink="http://www.w3.org/1999/xlink">
+                <circle cx="256" cy="256" fill="#20515b" id="ellipse" r="256" />
+                <path
+                  d="M372.4,168.7c0,0-33.3-26.1-72.7-29.1l-3.5,7.1c35.6,8.7,51.9,21.2,69,36.5  c-29.4-15-58.5-29.1-109.1-29.1s-79.7,14.1-109.1,29.1c17.1-15.3,36.5-29.2,69-36.5l-3.5-7.1c-41.3,3.9-72.7,29.1-72.7,29.1  s-37.2,54-43.6,160c37.5,43.3,94.5,43.6,94.5,43.6l11.9-15.9c-20.2-7-43.1-19.6-62.8-42.3c23.5,17.8,59.1,36.4,116.4,36.4  s92.8-18.5,116.4-36.4c-19.7,22.7-42.6,35.3-62.8,42.3l11.9,15.9c0,0,57-0.3,94.5-43.6C409.6,222.7,372.4,168.7,372.4,168.7z   M208.7,299.6c-14.1,0-25.5-13-25.5-29.1s11.4-29.1,25.5-29.1c14.1,0,25.5,13,25.5,29.1S222.8,299.6,208.7,299.6z M303.3,299.6  c-14.1,0-25.5-13-25.5-29.1s11.4-29.1,25.5-29.1s25.5,13,25.5,29.1S317.3,299.6,303.3,299.6z"
+                  fill="#f6f8f8" id="logo" /></svg>
```

Comment:
Can you please move this code to own reusable file like `partials/icons/discord.hbs`, and reuse them by `{{> icons/discord}}`